Database/Corrections/mopObjectFixes.lua

```diff
@@ -215,5 +215,15 @@ function MopObjectFixes.Load()
             [objectKeys.spawns] = {[zoneIDs.KRASARANG_WILDS] = {{51.58,33.08},{47.24,29.63},{47.30,30.53},{48.61,33.45},{46.66,36.47},{47.22,38.36},{46.59,38.07},{41.29,41.19},{41.98,42.24},{40.14,41.87},{39.16,39.98},{38.83,38.51},{37.11,39.04},{37.14,37.00},{37.89,34.22},{43.55,32.98}}},
             [objectKeys.zoneID] = zoneIDs.KRASARANG_WILDS,
         },
+        [223820] = { -- Items for Build Your Own Raft
+            [objectKeys.name] = 'Spare Plank', -- DB says "unk name" and wowhead does not know this object, so we reuse it.
```

Comment:
There are four objects:

https://www.wowhead.com/object=211376/spare-plank
https://www.wowhead.com/object=211377/spare-plank
https://www.wowhead.com/object=211378/spare-plank
https://www.wowhead.com/object=211379/spare-plank

I'd prefer to use these.